### PR TITLE
Feature/estimator performance

### PR DESCRIPTION
**Pull request by chrstrom**
Adds loops in main that calculates the variance for the estimated omega and phi, and compares to CRLB.

Also fixes the fast_dtft-dependencies in estimation.py, and enforces inputs for every config-variable that will change in between attempts.

### CODE REVIEW

**Review by oyssolbo — APPROVED** [review]
Ser bra ut

File: scripts/estimation.py [hunk 1]
```diff
@@ -74,16 +74,16 @@ def estimate_omega(self, x_d):
     return omega_hat
 
 
-  def estimate_phase(self, x_d):
+  def estimate_omega_and_phi(self, x_d):
     # Estimates the phase of the signal
```
Comment (oyssolbo) [inline scripts/estimation.py:77]: Kanskje endre kommentara litt no ;)

Comment (chrstrom) [reply scripts/estimation.py:77]: on it chief

**Review by oyssolbo — APPROVED** [review]
Sweet! Ser at han Sverre har hatt innverkning på deg

**Comment by chrstrom** [comment]
> Sweet! Ser at han Sverre har hatt innverkning på deg

Hææææææææææææææææææææææææææ

**Pull request merged**